deploy/deploy.sh

```
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

how about adding ```set -e``` ? (http://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#The-Set-Builtin)

---

+1
ignoring errors is impolite.